02week/pigLatin.js

```diff
@@ -7,13 +28,34 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+const vowels = ['a', 'e', 'i', 'o', 'u'];
 
-function pigLatin(word) {
-
-  // Your code here
 
+function pigLatin(word) {
+ 
+  const word = str.split('');
```

Comment:
This variable is attempting to redefine the argument word that's being passed into your pigLatin function.